book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
```

Comment:
- [ ] It would be better if in implementing the add_rental method, the book itself should be sent as a parameter to create the new rental achieving the `has-many` association. For example; `Rental.new(date, self, person)`

Reply:
Thanks, @addod19 Great tip.